datahub-web-react/src/app/homeV3/template/components/addModuleMenu/useAddModuleMenu.tsx

```diff
@@ -133,6 +141,7 @@ export default function useAddModuleMenu(
             onClick: () => {
                 handleOpenCreateModuleModal(DataHubPageModuleType.AssetCollection);
             },
+            disabled: isSmallModuleRow,
```

Comment:
can we add tooltips to disabled these disabled dropdown items? they can say "Cannot add large widget to small widget row" or vice versa for the other kinds.

We can come back to this later as well I think, since ideally we have the functionality discussed on slack where adding a mismatched module size to a row will just add the module to a new row right below